lib/load/stats.ex

```diff
@@ -8,14 +8,16 @@ defmodule Stats do
     last_ms: 0, # last time stats were collected
     requests: 0,
     succeeded: 0,
-    failed: 0
+    failed: 0,
+    latencies: []
```

Comment:
latencies were intentionally left out as the only latency that makes sense in a p2p traversal is the one emerging from the collector

Reply:
that said it helps to have them, for other use cases (i.e. test a web server)